homework1/task03/find_maximum_and_minimum.py

```diff
@@ -0,0 +1,31 @@
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+We guarantee, that file exists and contains line-delimited integers.
+To read file line-by-line you can use this snippet:
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+"""
+from typing import Tuple
+
+
+def scan(file_name: str) -> Tuple[int, int]:
+    with open(file_name) as fi:
+        lst = fi.readlines()

```

Comment:
never read the whole file into memory. In case of small files, it will be usually OK, but you neven know when you are going to be fed a large one. Python is not exactly memory-efficient.
Use iteration of lines through file:

     for line in fi: